.pre-commit-config.yaml

```diff
@@ -12,12 +12,14 @@ repos:
           '--indent', '4',
           '--no-sort-keys',
         ]
-      #    -   id: requirements-txt-fixer
       - id: trailing-whitespace
-  #-   repo: meta
-  #    hooks:
-  #    -   id: check-hooks-apply
-  #    -   id: check-useless-excludes
```

Comment:
This is only supported with `Pipfile` or `requirements.txt` and therefore was removed.